doc/releases/changelog-dev.md

```diff
@@ -201,6 +201,12 @@
 
 <h3>Deprecations 👋</h3>
 
+* `qml.operation.Observable` and the corresponding `Observable.compare` have been deprecated, as
+  pennylane now depends on the more general `Operator` interface instead. The
+  `Operator.is_hermitian` property can instead be used to check whether or not it is highly likely
+  that the operator instance is Hermitian.
```

Comment:
```suggestion
  that the operator instance is Hermitian. Please consult the :doc:`deprecations and removals page </development/deprecations>` for more information.
```

Reply:
Should we just leave that as a generic comment for all deprecations at the top of the Deprecations section?